### ISSUE TITLE
Python 3.8

### ISSUE THREAD

**Issue by NathanDeMaria** [quiltdata/quilt#1432]
`quilt3` won't install in an otherwise empty Python 3.8 environment, because of the `ruamel.yaml<=0.15.70` requirement's `setup.py` issue. That's fixed in `ruamel.yaml==0.15.78`  (see [here](https://github.com/ware/ruamel-yaml/commit/18fd2b9816616ae69ea1e2f1ed5cda90e8b7117f))

Repro:
```bash
conda create -n test-env python=3.8 pip -y
conda activate test-env
pip install quilt3
```

```
Collecting quilt3
  Using cached https://files.pythonhosted.org/packages/0e/c0/11cee00dc113cf3b5b3e4a1f42200234e4959bee550978708bf5ee6dbe33/quilt3-3.1.6-py3-none-any.whl
Collecting ruamel.yaml<=0.15.70
  Using cached https://files.pythonhosted.org/packages/1c/f7/e07ae5726bc24c06c130a13c78fe3a9ae07159d5fc94e1dc87bc3f3c303d/ruamel.yaml-0.15.70.tar.gz
    ERROR: Command errored out with exit status 1:
     command: 'C:\Users\nathan.demaria\AppData\Local\Continuum\miniconda3\envs\test-env\python.exe' -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\nathan.demaria\\AppData\\Local\\Temp\\pip-install-3kcgoy1m\\ruamel.yaml\\setup.py'"'"'; __file__='"'"'C:\\Users\\nathan.demaria\\AppData\\Local\\Temp\\pip-install-3kcgoy1m\\ruamel.yaml\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base 'C:\Users\nathan.demaria\AppData\Local\Temp\pip-install-3kcgoy1m\ruamel.yaml\pip-egg-info'
         cwd: C:\Users\nathan.demaria\AppData\Local\Temp\pip-install-3kcgoy1m\ruamel.yaml\
    Complete output (11 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\nathan.demaria\AppData\Local\Temp\pip-install-3kcgoy1m\ruamel.yaml\setup.py", line 211, in <module>
        pkg_data = _package_data(__file__.replace('setup.py', '__init__.py'))
      File "C:\Users\nathan.demaria\AppData\Local\Temp\pip-install-3kcgoy1m\ruamel.yaml\setup.py", line 184, in _package_data
        data = literal_eval("".join(lines))
      File "C:\Users\nathan.demaria\AppData\Local\Temp\pip-install-3kcgoy1m\ruamel.yaml\setup.py", line 156, in literal_eval
        return _convert(node_or_string)
      File "C:\Users\nathan.demaria\AppData\Local\Temp\pip-install-3kcgoy1m\ruamel.yaml\setup.py", line 95, in _convert
        if isinstance(node, Str):
    NameError: name 'Str' is not defined
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

FWIW, I'm not sure why there's a version ceiling on `ruamel.yaml` here, or if that's the only issue with `quilt3`+Python 3.8, so I'm running locally with Python 3.8 + latest `ruamel.yaml` now and will drop a note if there's any issues

**Comment by akarve** [comment]
Thanks. We'll take a look.

**Comment by akarve** [comment]
Fixed in #1440

**Comment by akarve** [comment]
Note that we still await a 3.8 compatible version of pyarrow to run quilt3 in 3.8
